src/app/pages/receive-tokens/receive-btc.tsx

```diff
@@ -12,10 +12,13 @@ import { ReceiveBtcModalWarning } from './components/receive-btc-warning';
 import { ReceiveTokensLayout } from './components/receive-tokens.layout';
 
 export function ReceiveBtcModal() {
-  const accountIndex = useCurrentAccountIndex();
-  const activeAccountBtcAddress = useBtcNativeSegwitAccountIndexAddressIndexZero(accountIndex);
   const analytics = useAnalytics();
   const { state } = useLocation();
+
+  const currentAccountIndex = useCurrentAccountIndex();
+  const accountIndex = get(state, 'accountIndex', currentAccountIndex);
```

Comment:
We need to be careful with this, as it can introduce many typescript `any` types into the codebase.

Better to wrap in a hook, and throw errors to narrow the type so by the time we consume it in the component it is what we expect